packages/calcite-components/src/components/block/block.tsx

```diff
@@ -86,7 +90,14 @@ export class Block
   /** When `true`, interaction is prevented and the component is displayed with lower opacity. */
   @property({ reflect: true }) disabled = false;
 
-  /** When `true`, displays a drag handle in the header. */
+  /** When `true`, the component is not draggable. */
```

Comment:
Nitpick: Maybe (?) this could use more context. "When `true`, and a parent Block Group is `drag-enabled`, the component is not draggable" - or something along those lines.

Reply:
Will add that doc context. cc @geospatialem

Reply:
Awesome addition! For consistency can we specify the component with the `calcite-`naming convention, so:

```
When `true` and a parent `calcite-block-group` is `dragEnabled`, the component is not draggable.
```